tests/test_datapackage.py

```diff
@@ -345,6 +345,44 @@ def test_raises_if_remote_resource_url_doesnt_exist(self):
         with pytest.raises(IOError):
             datapackage.DataPackage(descriptor).resources[0].data
 
+    @httpretty.activate
+    def test_if_relative_resource_of_remote_package_has_absolute_path(self):
```

Comment:
Yes, ultimately it does. I decided to added it in case someone decides that `local_data_path` shouldn't be `None`. Also, this is how interpreted your remark in #97:

> Just make sure write a failing test that triggers this bug before actually solving it, so we're sure it doesn't creep back in the code.

I'll gladly remove it though, the less code the better.